instance/models/instance.py

```diff
@@ -134,6 +152,68 @@ def status(self):
         else:
             return active_server_set[0].status
 
+    @property
+    def event_context(self):
+        """
+        Context dictionary to include in events
+        """
+        return {'instance_id': self.pk}
+
+    @staticmethod
+    def on_post_save(sender, instance, created, **kwargs):
+        """
+        Triggered by the post_save event, with `created` a boolean indicating if the object was just
+        created in the DB
+        """
+        self = instance
+        self.update_logger()
```

Comment:
@mtyaka Yup, that was what I first tried, but there seem to be some argument manipulation going on, Django was choking on a missing instance attribute when I changed it there. If you see something else I can try to avoid the `self = instance` line, I'd be happy to - I'm not particularly happy about it either.